src/main/java/baritone/process/BuilderProcess.java

```diff
@@ -532,7 +546,7 @@ public int lengthZ() {
         }
 
         Optional<Tuple<BetterBlockPos, Rotation>> toBreak = toBreakNearPlayer(bcc);
-        if (toBreak.isPresent() && isSafeToCancel && ctx.player().isOnGround()) {
+        if (toBreak.isPresent() && isSafeToCancel && ctx.player().onGround()) {
```

Comment:
Is there a reason for this change? It looks unrelated.

Reply:
oh ups no maybe this happend because i tested it in 1.20.1 build